CenterForOpenScience/osf.io
# [ENG-1917] Shift preprintprovider views for maintainability

## Purpose

When updating Django we exposed some bugs within the admin views of preprint provider code.

## Changes

Some elements on the form are moved to separate template pages. I try to split up these views and use a format of buttons and single page form, like we use for the user admin page. 

<img width="1171" alt="Screen Shot 2020-05-31 at 5 23 18 PM" src="https://user-images.githubusercontent.com/9688518/83362949-72b86b80-a363-11ea-8716-ff9c9c4b1786.png">

UI changes simply take the custom taxonomy view and modify attribute view and make them a their own smaller, quicker loading pages.

## QA Notes

Make sure all preprint provider admin functions are intact and demonstrate adequate UI. Add a customs taxonomy, modify license etc.

## Documentation

technical task.

## Side Effects



## Ticket
